conda_lockfiles/loaders/records_from_urls.py

```diff
@@ -0,0 +1,59 @@
+from typing import Any
+
+from conda.base.context import context
+from conda.core.package_cache_data import PackageCacheData, ProgressiveFetchExtract
+from conda.exceptions import (
+    CondaExitZero,
+    DryRunExit,
+)
+from conda.models.match_spec import MatchSpec
+from conda.models.records import PackageRecord
+
+CondaPackageURL = str
+CondaPackageMetadata = dict[str, Any]
+
+
+def records_from_urls(
```

Comment:
Since this is specific to conda urls should we name the function accordingly?

```suggestion
def records_from_conda_urls(
```